contrib/helloimage/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker://docker.io/library/alpine:latest
+
+WORKDIR /tmp
```

Comment:
not needed, if you use absolute paths

Reply:
Thanks, I think I'll leave it, it's a bit of a security blanket for me.